frontend/src/components/navigation/Header.css

```diff
@@ -0,0 +1,32 @@
+.header-bar {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 20px 0px 20px 0px;
+}
+
+.header-information {
+    align-items: center;
+    background-color: #fafafa;
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    /* TODO: this width changes on toggle?? */
```

Comment:
Looks like the `homepage-display-state-toggle-section` is also changing width on toggle 😳